samples/attested_tls/client/tls_client.edl

```diff
@@ -2,8 +2,14 @@
 // Licensed under the MIT License.
 
 enclave {
-    from "openenclave/edl/logging.edl" import *;
-    from "openenclave/edl/syscall.edl" import *;
+    from "openenclave/edl/fcntl.edl" import
+        oe_syscall_read_ocall,
+        oe_syscall_write_ocall,
+        oe_syscall_close_ocall,
+        oe_syscall_writev_ocall,
+        oe_syscall_dup_ocall;
```

Comment:
Since this is a sample, we need to educate the developer.  How was this list obtained?   These aren't APIs that the sample directly calls, right?  So how does the developer figure out that this is the correct list?   Add a // comment explaining how this list was determined.   For example, do you take the APIs that the code calls and look them up in some doc that gives you things to put on this list?   If so, where is that doc?

Reply:
@dthaler , we have this issue to address this; https://github.com/openenclave/openenclave/issues/2719